docs/source/command_line.rst

```diff
@@ -575,26 +575,24 @@ of the above sections.
 .. option:: --local-partial-types
 
     In mypy, the most common cases for partial types are variables initialized using ``None``,
-    but without explicit ``Optional`` annotations. By default, mypy won't check partial types
+    but without explicit ``... | None`` annotations. By default, mypy won't check partial types
```

Comment:
Could using `...` here be interpreted as talking about an actual Ellipsis literal in the annotation? It's not legal in this case, but there are other cases where `...` has a specific meaning, so there's a risk of confusion.

Maybe something like `X | None` would be clearer?

```suggestion
    but without explicit ``X | None`` annotations. By default, mypy won't check partial types
```